src/setup_eval.R

```diff
@@ -0,0 +1,6 @@
+source(file.path("src", "write_config.R"))
+
+write_config(
+  noro_nowcast_dates = c("2023-12-10", "2024-01-21", "2024-02-25"),
```

Comment:
flagging that manually editting the config was given as the reason for needing a config. As before happy for you to go on with the approach that works for you

Reply:
yes I get that it just conflates with the logic for why configs are nice but its no big deal.